en/mulesoft/modules/ROOT/pages/ms_composer_build_a_flow_using_templates.adoc

```diff
@@ -13,4 +13,20 @@ Some examples of templates include:
 
 For more information about how to create a new flow from template, refer to xref:ms_composer_flows.adoc#create-a-new-flow-from-template[Create a New Flow From Template].
 
+== Customizing Your Template
+
+Since you will likely need to further customize your flow from a template, consider the following recommendations and guidelines when encountering messages while customizing your flow from template.
+
+=== Instance Requires Different Fields
+
+Your instance might have fields that are different from the template. For example, you might need to add a required field that is not used or delete a field used in the template that does not exist in your connection. These will be presented as errors that need to be resolved before testing the flow.
+
+=== Field Does Not Exist With Your Connection
```

Comment:
I suggest making the header the exact message that the user sees. Then you can start the next paragraph with "This message appears..." (I have the same suggestion for the next section as well.)

This is less repetitive, but it is up to you.